content/articles/servlet-request-and-servlet-collaboration/SevletRequest and ServletCallaboration.md

```diff
@@ -68,12 +68,16 @@ The `ServletRequest` interface provides us with many methods to work wi
 - `removeAttribute(String name)`: Used when removing the attribute from the current request of a type `void.`
 
 ### ServletRequest interface example
-We will create a user login project with a servlet in this example. In this context, we have been using `getParameter()` to return the value of the specified query parameter name.
+We will create a user login project with a servlet in this example. In this context, we will be using `getParameter()` to return the value of the specified query parameter name.
+
 **Steps to create this project.**
 
 Step 1: Open Eclipse IDE for Java EE developers. Create a Dynamic web project by selecting files, new then Dynamic web projects, and press enter.

```

Comment:
format your steps such that they are bold for visibility ie: 

**Step 1:** Open Eclipse IDE for Java EE developers. Create a Dynamic web project by selecting files, new then Dynamic web projects, and press enter.

please do so for all the other steps